yahoo_fantasy_api/yhandler.py

```diff
@@ -288,7 +288,7 @@ def put_transaction(self, transaction_key, xml):
         return self.put("transaction/" + str(transaction_key), xml)
 
     def get_player_stats_raw(self, game_code, player_ids, req_type, date,
```

Comment:
This is a mocked version of this function that should be updated too in `tests/mock_yhandler.py`

Reply:
I added the week param to the function here. Let me know if there's anything else you'd like me to add!